api/fastify-app.js

```diff
@@ -0,0 +1,70 @@
+import Fastify from 'fastify'
+import * as Sentry from '@sentry/node'
+import { createRoutes } from './routes/index.js'
+
+export async function createFastifyApp({
+  pgPool,
+  logger,
+  dealIngestionAccessToken,
+  domain
+}) {
+  const fastify = Fastify({
+    logger: {
+      level: 'info',
+      transport: {
+        target: 'pino-pretty'
+      }
+    }
```

Comment:
```suggestion
    logger
```

To allow the caller of `createFastifyApp` to provide the logger

Reply:
Alright, noted